api-samples/printing/printers.js

```diff
@@ -14,16 +20,34 @@ function onPrintButtonClicked(printerId, dpi) {
         horizontal_dpi: dpi.horizontal_dpi,
         vertical_dpi: dpi.vertical_dpi
       },
-      media_size: {
-        width_microns: 210000,
-        height_microns: 297000,
-        vendor_id: 'iso_a4_210x297mm'
-      },
       collate: { collate: false }
     }
   };
 
-  fetch('test.pdf')
+  if (rollPrintingEnabled()) {
+    filename = 'test-rollprinting.pdf';
+    ticket.print.media_size = {
+      width_microns: 72320,
+      // Note that this value needs to be between min_height_microns and
+      // max_height_microns.  Usually this matches the height of the document
+      // being printed.
+      height_microns: 110000
+    };
+    // This only makese sense to specify if the printer supports the trim
+    // option.
+    if (performTrim) {
```

Comment:
It can be specified any time the printer supports that option (so, it's dependent on the printer's capabilities).  However, it only makes sense for roll printing.